### PR TITLE
Fix Cartographic Composer wxPython deprecation warning

### PR DESCRIPTION
**Pull request by tmszi**
Fix Cartographic Composer wxPython deprecation warning: `wxPyDeprecationWarning: Call to deprecated item. Use SetToolTip instead.`

### CODE REVIEW

**Comment by tmszi** [comment]
Alright, but do you mean this [PR](https://github.com/OSGeo/grass/pull/310)?

**Comment by petrasovaa** [comment]
No, I mean this one. Something is unclear?

File: gui/wxpython/psmap/dialogs.py [hunk 1]
```diff
@@ -2512,26 +2504,21 @@ def _ColorsLinePanel(self, notebook):
         self.outlineCheck = wx.CheckBox(
```
Comment (petrasovaa) [inline gui/wxpython/psmap/dialogs.py]: Please use the CheckBox from wrap.py here.

File: gui/wxpython/gui_core/wrap.py [hunk 2]
```diff
@@ -122,6 +124,22 @@ def __init__(self, *args, **kwargs):
         wx.SpinCtrl.__init__(self, *args, **kwargs)
 
 
+class FloatSpin(fs.FloatSpin):
```
Comment (petrasovaa) [inline gui/wxpython/gui_core/wrap.py]: Looks good, but please add also the SetToolTip function as in other classes to the FloatSpin class to keep it consistent. The wrappers were originally developed for compatibility between wx3 and 4 and although that may not be such a big concern now, it's generally good idea to have wrappers around GUI library for various OS/versions compatibility issues.

File: gui/wxpython/psmap/dialogs.py (hunk 1 above)
Comment (tmszi) [reply gui/wxpython/psmap/dialogs.py]: Okay I used.

File: gui/wxpython/gui_core/wrap.py (hunk 2 above)
Comment (tmszi) [reply gui/wxpython/gui_core/wrap.py]: Okay I added some wrapper classes.

**Comment by petrasovaa** [comment]
Thank you, that's actually more changes than I asked for. There are still some issues when testing the composer, but I will address them in separate PR.

**Pull request merged**